tests/all/comments.rs

```diff
@@ -0,0 +1,56 @@
+use super::project;
+
+#[test]
+fn works() {
+    let mut p = project();
+    p.file("src/lib.rs", r#"
+            #![feature(proc_macro, wasm_custom_section, wasm_import_module)]
+
+            extern crate wasm_bindgen;
+
+            use wasm_bindgen::prelude::*;
+
+            #[wasm_bindgen]
+            /// This comment should exist
+            pub fn annotated() -> String {
+                String::new()
+            }
+
+            #[wasm_bindgen]
+            pub struct Annotated {
+                /// This comment should not exist
+                a: String,
+                /// This comment should exist
+                pub b: u32
+            }
+
+            #[wasm_bindgen]
+            impl Annotated {
+                #[wasm_bindgen(method)]
+                /// This comment should exist
+                pub fn get_a(&self) -> String {
+                    self.a.clone()
+                }
+            }
+        "#);
+
+        let (root, target) = p.build();
+        p.gen_bindings(&root, &target);
+        let js = p.read_js();
+        let comments = extract_doc_comments(&js);
+        /// 
```

Comment:
Was there supposed to be something here?

Reply:
nope, not sure how this got there, it has been removed. I also updated the test to handle white-space a little better